### ISSUE TITLE
about the training codes.

### ISSUE THREAD

**Issue by fuxuliu** [sciencefans/trojans-face-recognizer#2]
Hi, could you share the training codes, like the new loss function and others, thank you so much.

**Comment by liuyuisanai** [comment]
Hi @Gary-Deeplearning , we will release the training code and the loss function code after the release 
 of a correlative publication.
I will leave this issue open and notice you again after we make an update. stay tuned :)

**Comment by fuxuliu** [comment]
@sciencefans yep, thanks for your reply.

**Comment by liuyuisanai** [comment]
@Gary-Deeplearning We've update the ```loss.py``` for the training loss.
It includes two loss functions, the original ```arcface``` and the proposed ```ArcNegFace```.
You can run and compare the results.
Thanks.

**Comment by fuxuliu** [comment]
@sciencefans I got it, thank you.

**Comment by clhne** [comment]
Great jobs!
But when release the training code?
